tests/build_memory_index.cpp

```diff
@@ -52,7 +52,7 @@ int build_in_memory_index(const diskann::Metric &metric, const std::string &data
         convert_labels_string_to_int(label_file, labels_file_to_use, mem_labels_int_map_file, universal_label);
         if (universal_label != "")
         {
-            LabelT unv_label_as_num = std::stoul(universal_label);
+            LabelT unv_label_as_num = 0;
```

Comment:
What is the change for?

Reply:
The universal_label input can be a string and this conversion to int doesnot look correct.
Ideally, we should load the label map and find int label corresponding to the universal label. Should I do that ?